Whiskey/Functions/Invoke-WhiskeyNUnit2Task.ps1

```diff
@@ -46,7 +46,10 @@ function Invoke-WhiskeyNUnit2Task
     {
         $reportGeneratorVersionArg['Version'] = $TaskParameter['ReportGeneratorVersion']
     }
-
+    
+    $openCoverArgs = $TaskParameter['OpenCoverArgument']

```

Comment:
These should be arrays of values that get passed to the command as-is. Using splatting, they can only pass/use in parameters that begin with `-`. 

Pass the args directly to the commands without splatting. 

When displaying them as verbose messages, separate them by spaces, e.g. `$openCoverArgs -join ' '`.